src/content/1.7/basics/keeping_up-to-date/migration.md

```diff
@@ -0,0 +1,678 @@
+---
+title: Migration
```

Comment:
I think we can change this to:

- title: "How to migrate PrestaShop 1.6 to PrestasShop 1.7
- menuTitle: "Migration"

Reply:
I would not say where we can migrate from, as it could be used with the older versions as well.

Reply:
I know but I'm thinking about SEO